docs/pages/components/Stepper.mdx

```diff
@@ -64,7 +64,7 @@ For more compact steppers, the additional descriptive information can be include
 
 <ComponentProps
   children={{
-    description: 'children to be rendered as the step\'s visible label (if not visible label is desired, use the "tooltip" prop instead)'
+    description: 'children to be rendered as the step\'s visible label (if a not visible label is desired, use the "tooltip" prop instead)'
```

Comment:
```suggestion
    description: 'children to be rendered as the step\'s visible label (if no visible label is desired, use the "tooltip" prop instead)'
```

I think we probably meant to do this?

Reply:
I can switch it back if that's the case!